package.json

```diff
@@ -53,6 +53,7 @@
         "onCommand:cosmosDB.createGraph",
         "onCommand:cosmosDB.connectMongoDB",
```

Comment:
In the PR description, change `Fix to issue #492` to `Fixes #492`. After you make that change, you should see a tootip that GitHub will automatically close this issue when you merge the PR. Like this:
![Screen Shot 2020-02-04 at 10 25 11 AM](https://user-images.githubusercontent.com/11282622/73774372-b7772180-4738-11ea-8202-f9baae4f52bf.png)